#Header header 
float32[6] angle # degrees
float32[6] vel # deg/s
#time armReadTimestamp 
# time startSyncTime 
# uint32 isrTicks 
int32 msg_rcv_ctr
int32 buffer_health